packages/utils/src/types/agent.utils.ts

```diff
@@ -0,0 +1,10 @@
+import type { Identity } from "@dfinity/agent";
+
+export interface CreateAgentParams {
```

Comment:
It can be a type I think, WDYT?

```
export type CreateAgentParams = Pick<HttpAgentOptions, "retryTime" | "verifyQuerySignatures" | "host" > & {identity: Identity; fetchRootKey?: boolean;}
```

PS: I guess `verifyQuerySignatures` is now the default so we can ignore the comment

Reply:
For sure! Done